internal/pkg/scorecard/helpers/versions.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schelpers
+
+import (
+	"fmt"
+	"github.com/spf13/viper"
+	"strings"
+)
+
+const V1alpha1 = "v1alpha1"
```

Comment:
These don't need to be exported, unless you pan on using them outside of this package.

Reply:
thanks, will fix this and resubmit the PR.